src/array_buffer.rs

```diff
@@ -544,4 +544,27 @@ impl ArrayBuffer {
       deleter_data,
     ))
   }
+
+  #[inline(always)]
+  /// Create a cheap, zero-sized backing store.
+  pub fn new_empty_backing_store() -> UniqueRef<BackingStore> {
+    // A zero-sized-but-valid, mutable buffer
+    static mut BUFFER: [u8; 0] = [];
+    // A function that does nothing
+    extern "C" fn empty_deleter(
+      _ptr: *mut c_void,
```

Comment:
**suggestion:** Expose the [EmptyDeleter](https://v8.github.io/api/head/classv8_1_1BackingStore.html#acb89ca23fdc981dd2b55cc6725ea4722) from V8.

Reply:
Ah yeah, so apparently this is actually an optimization that V8 uses internally. We should be using this static value. But also, we should be exposing `EmptyBackingStore` itself.